management_api.go

```diff
@@ -58,6 +58,19 @@ func (e *Enforcer) GetAllRoles() []string {
 	return e.model.GetValuesForFieldInPolicyAllTypes("g", 1)
 }
 
+// GetAllRolesInDomain gets the list of roles in domain that show up in the current policy.
+func (e *Enforcer) GetAllRolesInDomain(domain string) []string {
```

Comment:
@peng19940915 should be ``GetAllRolesInDomain(domain string, ptype string...)``. ptype is determined by the user, and if the user does not pass in ptype, to get all by default.